extensions/scarb-mdbook/src/main.rs

```diff
@@ -0,0 +1,46 @@
+use anyhow::{Context, Result};
+use camino::Utf8PathBuf;
+use clap::Parser;
+use mdbook::MDBook;
+use scarb_ui::Ui;
+use scarb_ui::args::VerbositySpec;
+use std::process::ExitCode;
+
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
+struct Args {
+    /// Path to book source directory.
+    #[arg(long)]
+    pub input: Utf8PathBuf,
+    /// Path to book output directory.
+    #[arg(long)]
```

Comment:
```suggestion
    #[arg(long, default_value_t = Utf8PathBuf::from("./book/"))]
```

Reply:
I see you added the default `/book/` output dir later in commits in `scarb doc --build` so the comment above is not as relevant. I think I'd rather still have a default output dir here but no strong feelings 🤔 so I'll just leave it for you to consider.